src/main/webapp/app/exercises/quiz/manage/quiz-question-list-edit-existing.component.ts

```diff
@@ -305,12 +361,20 @@ export class QuizQuestionListEditExistingComponent implements OnChanges {
                     dropLocation.invalid = false;
                 });
                 for (const dragItem of dndQuestion.dragItems || []) {
+                    var dragItemCounter = 0;
```

Comment:
Please don't use var, use let instead. For me this declaration seems like it should be outside of the loop.
```suggestion
                let dragItemCounter = 0;
                for (const dragItem of dndQuestion.dragItems || []) {
```

Reply:
yes it should. i just fixed it